docs/PUBLISHING.md

```diff
@@ -92,14 +92,14 @@ access resin.io's production downloads S3 bucket.
 Run the following command to publish a specific file:
 
 ```sh
-./scripts/publish/aws-s3.sh <file>
+./scripts/publish/aws-s3.sh -f <file> -b <bucket> -v <version> -t <production|development>
```

Comment:
I'm trying to not hardcode anything at all anywhere. I propose to decouple `bintray-debian.sh` from `etcher` instead.